test/fixtures/Enquiry.js

```diff
@@ -0,0 +1,34 @@
+const faker = require('faker');
+
+exports.Name = { 
+    message: 'OK',
+    data:
+   { 
+     message: 'success',
+     response: 'success',
+     data:
+      { 
+        AccountName: 'John Doe',
+        sessionID: '999232200107170915323048583333',
+        AccountNumber: ' 0037514056',
+        status: '97',
+        BVN: '12345678901',
+        ResponseText: null 
+      } 
+    } 
+  }
+
+exports.data = {
+    host:`https://innovation-sandbox-backend.herokuapp.com`,
+   sandbox_key: '0ae0db703c04119b3db7a03d7f854c13',
+   params:{
+    Referenceid: "01",
+    RequestType: "01",
+    Translocation: "01",
+    ToAccount: "0037514056",
+    destinationbankcode: "000001"
+   },
+   subscription_key: "t",
+   Appid: 69,
+   ipval: 0
+}
```

Comment:
Use random mock data and not actual data

Reply:
noted